net/bulk_emulation/mem_xprt_xo.c

```diff
@@ -46,6 +46,9 @@ static const struct m0_net_bulk_mem_ops mem_xprt_methods;
 M0_INTERNAL int m0_mem_xprt_init(void)
 {
 	m0_list_init(&mem_domains);
+	m0_net_xprt_register(&m0_net_bulk_mem_xprt);
+	if (m0_net_xprt_default_get() == NULL)
+		m0_net_xprt_default_set(&m0_net_bulk_mem_xprt);
```

Comment:
Maybe setting bulk_mem as default is not necessary.
Because it is used only for emulation purpose.

Reply:
bulk mem transport will be set as default only if no other transport is set as default.